docs/USAGE.md

```diff
@@ -55,6 +56,35 @@ There are a number of strategies available to provide [secure remote access]({{b
 
 If you want to use openHAB Android on a wall mounted tablet, go to settings and select `Disable display timer` and `Fullscreen`.
 
+## Alarm Clock
+
+The openHAB app will send the next wake-up time from your alarm clock app to the server. Please configure an Item name in the settings.
```

Comment:
Probably should mention the format used for sending  (`The time is sent as a number containing the number of milliseconds since the epoch` or something like that)